sound/soc/sof/ipc3-topology.c

```diff
@@ -2246,6 +2246,18 @@ static int sof_ipc3_tear_down_all_pipelines(struct snd_sof_dev *sdev, bool verif
 	list_for_each_entry(sroute, &sdev->route_list, list)
 		sroute->setup = false;
 
+	/*
+	 * before suspending, make sure the refcounts are all zeroed out. There's no way
```

Comment:
this is not going to make sure that the refcounts reached zero, but checks it for reporting it.